lms/envs/common.py

```diff
@@ -2885,6 +2888,21 @@
 # Dafault site id to use in case there is no site that matches with the request headers.
 DEFAULT_SITE_ID = 1
 
+# Settings for Comprehensive Theming app
+
+# Cache time out settings
+# by Comprehensive Theme system
+THEME_CACHE_TIMEOUT = 30 * 60
```

Comment:
It would be great if there was an easier way to control this value, but I'm not sure it's critical for this particular PR